test/functional/ui/inccommand_spec.lua

```diff
@@ -805,6 +805,7 @@ describe(":substitute, inccommand=split", function()
   it('does not show split window for :s/', function()
     feed("2gg")
     feed(":s/tw")
+    screen:sleep(1)
```

Comment:
There's nothing directly wrong with it, but I'd be surprised if sleeping 1ms makes such a big difference.